pkg/virt-launcher/virtwrap/manager_test.go

```diff
@@ -491,7 +498,6 @@ var _ = Describe("Manager", func() {
 		})
 		It("should verify that migration failure is set in the monitor thread", func() {
 			isMigrationFailedSet := make(chan bool, 1)
-
```

Comment:
Why did you remove the empty line?

Reply:
due to iterations mistake, 
i saw it , didn't want to push yet because wanted the test to run.